fdbclient/vexillographer/fdb.options

```diff
@@ -262,6 +262,8 @@ description is not currently required but encouraged.
             description="This option should only be used by tools which change the database configuration." />
     <Option name="report_conflicting_keys" code="712"
             description="The transaction can retrieve keys that are conflicting with other transactions." />
+    <Option name="special_key_space_relaxed" code="713"
+            description="By default, the special key space will only allow users to read from exactly one module. Use this option to allow reading from zero or more modules. Users who set this option should be prepared for new modules, which may have different behaviors than the modules they're currently reading. For example, a new module might block or return an error." />
```

Comment:
Can we explain what module is here? For example, a module is a sub key space in the special key space.

Reply:
Done, thanks